src/unix/linux-core.c

```diff
@@ -107,6 +109,26 @@ int uv__platform_loop_init(uv_loop_t* loop) {
 }
 
 
+int uv__platform_loop_init(uv_loop_t* loop) {
+  return uv__epoll_init(loop, 1);
+}
+
+
+int uv__io_fork(uv_loop_t* loop) {
+  int err;
+  uv__close(loop->backend_fd);
+  if (loop->inotify_fd != -1) {
+    uv__io_stop(loop, &loop->inotify_read_watcher, POLLIN);
+    uv__close(loop->inotify_fd);
+    loop->inotify_fd = -1;
+    err = uv__init_inotify(loop);
+    if (err)
+      return err;
+  }
+
+  return uv__epoll_init(loop, 0);
```

Comment:
We never made any promises about the background fd not changing,ww have an API for it, so we can document this behavior and make it official. I'd like to follow this approach.